.gitignore

```diff
@@ -144,3 +144,4 @@ src/stcal/_version.py
 # auto-generated API docs
 docs/source/api
 .DS_Store
+.DS_Store
```

Comment:
This ignore is already in the line above, so your addition is a duplicate.  I'm not sure why git didn't previously ignore the `.DS_Store`.